internal/kafka/command_cluster_describe.go

```diff
@@ -35,8 +35,8 @@ type describeStruct struct {
 	Network            string `human:"Network,omitempty" serialized:"network,omitempty"`
 	Status             string `human:"Status" serialized:"status"`
 	Endpoint           string `human:"Endpoint" serialized:"endpoint"`
-	ByokKeyId          string `human:"BYOK Key ID" serialized:"byok_key_id"`
-	EncryptionKeyId    string `human:"Encryption Key ID" serialized:"encryption_key_id"`
```

Comment:
These are both external resources (outside of Confluent). There's a big difference between the "BYOK key" and "BYOK key ID" and both are strings. We might want to re-evaluate this change...

Reply:
okay, let me revert this one back. Any others that need to still contain ID?

Reply:
A general rule of thumb would be anything containing "Azure", "Aws", or "Gcp" should keep the "ID". For instance, `AzurePrivateLinkServiceId` should stay as is

Reply:
or `AwsVpcEndpoint`

Reply:
got it , reverting these ones